### PR TITLE
Fixing conversationState in listener

### PR DESCRIPTION
**Pull request by sayachi**

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java [hunk 1]
```diff
@@ -91,7 +91,6 @@ public void postSave(Membership m, boolean isNew) throws Exception {
       Space space = spaceService.getSpaceByGroupId(m.getGroupId());
       //TODO A case to confirm: will we create a new space here when a new group is created via organization portlet
       if (space != null) {
-        space.setEditor(ConversationState.getCurrent().getIdentity().getUserId());
```
Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]: Can you explain your change please ? Are you sure this attribute is not used somewhere else ?

Comment (sayachi) [inline component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]: This line  
`space.setEditor(ConversationState.getCurrent().getIdentity().getUserId());`
must not retrieve the identity of the connected user, so, it shouldn't be present on a listener.

Comment (thomasdelhomenie) [inline component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java]: That's not my question. I agree ConversationState.getCurrent().getIdentity().getUserId() should not be called here, but by removing this line, the editor of the space is not set anymore. We must be sure it has no impact. Did you check ?

**Comment by sayachi** [comment]
Merged in develop

**Pull request closed without merging**